lib/recurly/adjustment.php

```diff
@@ -60,7 +60,7 @@ public function delete() {
    * @return Recurly_Invoice the new refund invoice
```

Comment:
`* @param String indicates the refund order to apply, valid options: {'credit_first', 'transaction_first'}, defaults to 'credit_first'`

Reply:
@visualex I made a quick adjustment to the documentation.

Reply:
@visualex when you add the changes I commented above to your PR, we'll go ahead and merge. Thanks!